style.css

```diff
@@ -0,0 +1,4 @@
+.padding {
```

Comment:
Try looking into the concept of a wrapper which can be placed as a class on your header, main and footer instead.